src/avfoundation.cs

```diff
@@ -12392,6 +12393,10 @@ interface AVPlayerItem : NSCopying {
 		[TV (15, 0), NoWatch, NoMac, NoiOS, NoMacCatalyst]
 		[Export ("translatesPlayerInterstitialEvents")]
 		bool TranslatesPlayerInterstitialEvents { get; set; }
+
+		[Watch (9, 0), TV (16, 0), Mac (13, 0), iOS (16, 0)]
+		[NullAllowed, Export ("nowPlayingInfo", ArgumentSemantic.Copy)]
+		NSDictionary<NSString, NSObject> NowPlayingInfo { get; set; }
```

Comment:
Looks like there is a way to make a smart dictionary from the MPNowPlayingInfoCenter properties, that said you can do something like and file an issue for this so we can have a look at this later

```suggestion
		NSDictionary WeakNowPlayingInfo { get; set; }
```